### PR TITLE
Create a directory to keep all the overview notebooks.

### PR DESCRIPTION
**Pull request by alarshi**
This PR adds an overview directory which contains the first notebook (overview) of each of the modules.

### CODE REVIEW

**Comment by ljhwang** [comment]
Great! Thanks for the additional sections and summary.  

Will the table of contents have hyperlinks to the section?

Please also correct multiple)(2) typos in the "tranfer" == "transfer" in the Objectives.

**Comment by alarshi** [comment]
@ljhwang : Thanks for the review!
I fixed the typos in the latest commit.

Unfortunately, I don't think it is possible to add hyperlinks to section titles across different notebooks :-/ But, I can add the notebook links, what do you think?

**Comment by ljhwang** [comment]
I was just thinking about links to the notebooks note the sections.  

There are ways to include section links but I do not know if it works across notebooks:
https://sebastianraschka.com/Articles/2014_ipython_internal_links.html

File: bin/overview-modules/0_overview-heat-flow-module.ipynb [hunk 1]
```diff
@@ -0,0 +1,210 @@
+{
```
Comment (naliboff) [inline bin/overview-modules/0_overview-heat-flow-module.ipynb:1]: <blockquote>Upon completion of this module you would know how to use python to:</blockquote><p>Replace "would" with "will"?</p><p> </p><blockquote>An introduction to the conductive heat transfer mechanism within the Earth.</blockquote><p>Suggestion: "An introduction to conductive heat transfer within the Earth."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

**Comment by naliboff** [comment]
@alarshi - Looks good, just a few minor comments.

**Comment by alarshi** [comment]
@naliboff Thank you for the suggestions! I addressed them in the latest commit.
@ljhwang : I don't think we can add section references across notebooks :( . I just added the links to the notebooks, hopefully that is enough for our purposes.

File: bin/overview-modules/0_overview-heat-flow-module.ipynb (hunk 1 above)
Comment (naliboff) [inline bin/overview-modules/0_overview-heat-flow-module.ipynb:1]: <blockquote>An introduction to <strong>the </strong>conductive heat transfer mechanism within the Earth. Add bolded word (<strong>the</strong>).</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/6/discussion/#comment-1806896991'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

**Comment by naliboff** [comment]
@alarshi - Just one more minor comment, and then this will be ready to merge.

**Comment by alarshi** [comment]
@naliboff : Thanks, just fixed it! :)

**Review by naliboff — APPROVED** [review]

**Pull request merged**